command/testdata/git/test_config.yml

```diff
@@ -15,7 +15,7 @@ tasks:
 
 modules:
 - name: sample
-  repo: https://github.com/evergreen-ci/sample.git
+  repo: git@github.com:evergreen-ci/sample.git
```

Comment:
should we test both?

Reply:
I decided to not support the https format since no one is using it and also it makes parsing the string harder
Ideally the owner and repo would be separate fields

Reply:
In that case, does it make more sense to remove the URL entirely? I think if we only support ssh in modules then it makes it harder to get out of requiring ssh on the hosts.

Reply:
It's not that we're only supporting ssh - we're using this to parse the repo to clone (the actual clone is done using the method specified in the command)

Reply:
how even. that's not super clear

Reply:
added support to both